src/routes/users.test.js

```diff
@@ -0,0 +1,33 @@
+const request = require('../test/supertest');
+const User = require('../models/user');
+const emails = require('../emails');
+
+describe('users route tests', () => {
```

Comment:
```suggestion
describe('users routes', () => {
```

Plural for routes. Also, everything here should be tests, so I think that adding that word does not really necessary.